tests/test_llm.py

```diff
@@ -369,3 +372,82 @@ async def _request_fnc_call(
         pass
 
     return stream
+
+
+_HEARTS_RGBA_PATH = Path(__file__).parent / "hearts.rgba"
+with open(_HEARTS_RGBA_PATH, "rb") as f:
+    image_data = f.read()
+
+    _HEARTS_IMAGE_VIDEO_FRAME = VideoFrame(
+        width=512, height=512, type=VideoBufferType.RGBA, data=image_data
+    )
+
+_HEARTS_JPEG_PATH = Path(__file__).parent / "hearts.jpg"
+with open(_HEARTS_JPEG_PATH, "rb") as f:
+    _HEARTS_IMAGE_DATA_URL = (
+        f"data:image/jpeg;base64,{base64.b64encode(f.read()).decode()}"
+    )
+
+
+@pytest.mark.parametrize("llm_factory", LLMS)
+async def test_chat_with_image_data_url(llm_factory: Callable[[], llm.LLM]):
```

Comment:
Let's limit this test to OAI? I think they're the only one to support it

Reply:
i modified the anthropic plugin to support these (by parsing the URL and reconstructing it as an object) and it passes the tests. not sure what will happen with vertex, didn't check that one locally so will watch in CI

Reply:
TIL Anthropic supports images